massa-protocol-worker-2/src/sig_verifier.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+//! Optimized batch signature verifier
+
+use massa_hash::Hash;
+use massa_protocol_exports_2::ProtocolError;
+use massa_signature::{verify_signature_batch, PublicKey, Signature};
+use rayon::{prelude::ParallelIterator, slice::ParallelSlice};
+
+/// Limit for small batch optimization
+const SMALL_BATCH_LIMIT: usize = 2;
```

Comment:
This code (and this constant in particular) could use its own benchmark code

Reply:
Added a comment